css/style.css

```diff
@@ -0,0 +1,236 @@
+html,
+body {
+  font-family: "Montserrat", sans-serif;
+  margin: 0px;
+}
+
+.main-nav-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: aliceblue;
+}
+
+.logo {
+  width: 30px;
+  height: 30px;
+  margin-left: 10px;
+}
+
+.main-navbar li {
+  list-style-type: none;
+  margin-right: 10px;
+}
+
+.desktop-show {
+  display: none;
+}
+
+.mobile-searchbar-container {
+  width: 90%;
+  margin: 10px auto;
+}
+
+#site-search-mobile {
+  width: 100%;
+  height: 30px;
+}
+
+h1 {
+  text-align: center;
+  margin-top: 40px;
+  margin-bottom: 30px;
+}
+
+h2 {
+  text-align: center;
+}
+
+a {
+  text-decoration: none;
+}
+
+/*carousel part below*/
+.carousel {
+  margin-bottom: 50px;
+}
+
+.carousel-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.carousel-container button {
+  position: relative;
+  bottom: 70px;
+}
+
+.product-card-carousel {
+  width: 240px;
+  height: auto;
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-bottom: 30px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+}
+
+.img-container {
+  width: 240px;
+  height: 240px;
+  margin: 0 auto;
+}
+
+.img-container img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+
+.product-name-carousel {
+  font-size: 16px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.product-name-regular {
+  font-size: 16px;
+}
+
+.product-card-carousel p {
+  font-size: 14px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+/*product list related css below*/
+.product-list-container {
+  width: 88%;
+  margin: 0 auto 50px auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+  gap: 20px 10px;
+}
+.product-card-regular {
+  justify-self: center;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+}
+
+.product-card-regular:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+.data-container {
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.img-container-regular {
+  width: 150px;
+  height: 140px;
+}
+
+.img-container-regular img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+footer {
+  background-color: aliceblue;
+  height: 100px;
+  padding-top: 10px;
+}
+
+footer ul {
+  width: 300px;
+  height: 70%;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+footer ul li {
+  flex: 1 1 50%;
+}
+
+/*whole page media queries*/
+@media only screen and (min-width: 860px) {
+  .desktop-show {
```

Comment:
Consider splitting css into multiple files (utilities, components, etc...). This looks like an utility class. So maybe

```css
/* styles/utilities/display.css */
.desktop-show { display: none }
@media only screen and (min-width: 860px) { ... } 
```

You can use `@import` to add to `style.css` file ;)